README.md

```diff
@@ -16,19 +21,54 @@ For more information, please check [our wiki page](https://wiki.redenmc.com/Undo
 All blocks, block entities and entities are undoable.
 Undo is available only if that change is caused by a player, for example, placing blocks and commands.
 
+
+## Control the game tick (tick)
+Reden provides the `/tick <function>` command to control game ticks conveniently and quickly.
+
+### Function (the `<function>` parameter)
+- `freeze` freeze the game tick
+- `query` display the current TPS information
+- `rate <tps>` adjust TPS (default is `20.0`)
+- `sprint <1d|3d|60s|stop>` speed up the game
+- `step` execute one game tick forward (only works when the game is frozen)
+- `unfreeze` unfreeze the game tick
+- `back` go back to the previous tick  ***Planning***
+
+
 ## RVC
-> WIP
+> Work in progress, functions listed below are implemented
+
+![image](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/b2d5f6de-1860-496e-8980-b4a2cd3abe95)
+
+RVC(Redstone Version Control), provides change history and diff analysis for your machines based on git. Press `R+L` to open RVC screen.
+
+### Update, commit and push
+***Notice: DO NOT use hotkey `R+G` to login to Github [Issue #103](https://github.com/zly2006/reden-is-what-we-made/issues/103)***
 
-RVC, Redstone Version Control, machine history and diff analysis based on git.
+If your Reden account has been linked with GitHub(You can link it [here](https://redenmc.com/home/edit)), RVC can upload your machines to GitHub. 
 
-RVC can upload your machines to GitHub if you have bound your GitHub account at https://redenmc.com.
 RVC supports subregions and submodules, so you can use other people's work in your own creations.
 
+
+For the first-time use, you need to log in to your Reden account. After logging in, you will see the following:
+
+![image](https://github.com/ArthurZhou/reden-is-what-we-made/assets/89689293/f4bf3dee-9eca-4d0b-bada-82ced5fd6745)
```

Comment:
describe the image, dont use "image"

Reply:
also, dont use hash as the filename, ** describe the image**

Reply:
> also, dont use hash as the filename, ** describe the image**

github use hashes as url by default. I can't change that

Reply:
> > also, dont use hash as the filename, ** describe the image**
> 
> github use hashes as url by default. I can't change that

???????

Reply:
> > also, dont use hash as the filename, ** describe the image**
> 
> github use hashes as url by default. I can't change that

oh dont drop files into the input box, you should upload the files via git to somewhere such as `docs/img` and then use them in markdown with a relative path